<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Friends</class>
 <widget class="QDialog" name="Friends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="friendTable">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>261</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="friendsChatBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="frinedsChatEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="friendAddEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="friendAddbutton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Friend</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="friendRemoveEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>120</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="friendRemoveButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>120</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="friendErrorLabel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
